r-package/grf/R/average_treatment_effect.R

```diff
@@ -25,14 +42,42 @@
 #'
 #' @param forest The trained forest.
 #' @param target.sample Which sample to aggregate treatment effects over.
+#'               Note: Options other than "all" are only currently implemented
+#'               for causal forests.
 #' @param method Method used for doubly robust inference. Can be either
 #'               augmented inverse-propensity weighting (AIPW), or
-#'               targeted maximum likelihood estimation (TMLE).
+#'               targeted maximum likelihood estimation (TMLE). Note:
+#'               TMLE is currently only implemented for causal forests with
+#'               a binary treatment.
 #' @param subset Specifies subset of the training examples over which we
 #'               estimate the ATE. WARNING: For valid statistical performance,
 #'               the subset should be defined only using features Xi, not using
 #'               the treatment Wi or the outcome Yi.
+#' @param debiasing.weights A vector of length n (or the subset length) of debiasing weights.
+#'               If NULL (default) these are obtained via the appropriate doubly robust score
+#'               construction, e.g., in the case of causal_forests with a binary treatment, they
+#'               are obtained via inverse-propensity weighting.
+#' @param compliance.score Only used with instrumental forests. An estimate of the causal
+#'               effect of Z on W, i.e., Delta(X) = E[W | X, Z = 1] - E[W | X, Z = 0],
+#'               which can then be used to produce debiasing.weights. If not provided,
+#'               this is estimated via an auxiliary causal forest.
+#' @param num.trees.for.weights In some cases (e.g., with causal forests with a continuous
+#'               treatment), we need to train auxiliary forests to learn debiasing weights.
+#'               This is the number of trees used for this task. Note: this argument is only
+#'               used when debiasing.weights = NULL.
 #'
+#' @references Athey, Susan, and Stefan Wager. "Efficient policy learning."
+#'             arXiv preprint arXiv:1702.02896, 2017.
+#' @references Chernozhukov, Victor, Juan Carlos Escanciano, Hidehiko Ichimura,
+#'             Whitney K. Newey, and James M. Robins. "Locally robust semiparametric
+#'             estimation." arXiv preprint arXiv:1608.00033, 2016.
+#' @references Imbens, Guido W., and Joshua D. Angrist. "Identification and Estimation of
+#'             Local Average Treatment Effects." Econometrica 62(2), 1994.
+#' @references Robins, James M., and Andrea Rotnitzky. "Semiparametric efficiency in
+#'             multivariate regression models with missing data." Journal of the
+#'             American Statistical Association 90(429), 1995.
+#' 
+#' @importFrom stats coef lm predict var weighted.mean
```

Comment:
since `lm` is imported here it can be omitted from `best_linear_projection`'s import